packages/nouns-contracts/test/foundry/rewards/ProposalRewards.t.sol

```diff
@@ -459,6 +506,21 @@ contract ProposalRewardsEligibilityTest is BaseProposalRewardsTest {
             votingClientIds: votingClientIds
         });
     }
+
+    function test_canceledProposalsAreIneligible() public {
+        params.proposalEligibilityQuorumBps = 7000;
```

Comment:
I am confused as to why you're using 7000 here. looks like bidder has 8 votes and supply is 12 so bidder has 67% not 70%, meaning with this config they are below quorum, but I think you want them to have quorum and only be ineligible bc it was canceled no?

Reply:
I copied the 7000 value from a test above. with 7000 it's still valid because of flooring:
floor(12*7000/10000) = 8

Reply:
added comment: dbc71b8330b798079fa7b70d2f71a4c904135a00